xml/ceph_maintenance_updates.xml

```diff
@@ -33,6 +33,257 @@
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      whether &rados; cache tiering is enabled (and which mode)
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      whether pools are replicated or erasure coded, and which erasure code
+      profile plug-in and parameters are in use
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      how many hosts are in the cluster, and how many hosts have each type of
+      daemon
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      whether a separate OSD cluster network is being used
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      how many RBD pools and images are in the cluster, and how many pools have
+      RBD mirroring enabled
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      how many RGW daemons, zones, and zonegroups are present; which RGW
+      frontends are in use
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      aggregate stats about the &crushmap;, such as which algorithms are used,
+      how big buckets are, how many rules are defined, and what tunables are in
+      use
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    If you had telemetry enabled before 14.2.5, you will need to re-opt-in
+    with:
+   </para>
+<screen>&prompt.cephuser;ceph telemetry on</screen>
+   <para>
+    If you are not comfortable sharing device metrics, you can disable that
+    channel first before re-opting-in:
+   </para>
+<screen>
+&prompt.cephuser;ceph config set mgr mgr/telemetry/channel_device false
+&prompt.cephuser;ceph telemetry on
+</screen>
+   <para>
+    You can view exactly what information will be reported first with:
+   </para>
+<screen>
+&prompt.cephuser;ceph telemetry show        # see everything
+&prompt.cephuser;ceph telemetry show device # just the device info
+&prompt.cephuser;ceph telemetry show basic  # basic cluster info
+</screen>
+  </listitem>
+  <listitem>
+   <para>
+    <emphasis role="bold">New OSD daemon command
+    <command>dump_recovery_reservations</command></emphasis>. It reveals the
+    recovery locks held (in_progress) and waiting in priority queues. Usage:
+   </para>
+<screen>
+&prompt.cephuser;ceph daemon osd.<replaceable>ID</replaceable> dump_recovery_reservations
+</screen>
+  </listitem>
+  <listitem>
+   <para>
+    <emphasis role="bold">New OSD daemon command
+    <command>dump_scrub_reservations</command>. </emphasis> It reveals the
+    scrub reservations that are held for local (primary) and remote (replica)
+    PGs. Usage:
+   </para>
+<screen>
+&prompt.cephuser;ceph daemon osd.<replaceable>ID</replaceable> dump_scrub_reservations
+</screen>
+  </listitem>
+  <listitem>
+   <para>
+    <emphasis role="bold">RGW now supports S3 Object Lock set of
+    APIs.</emphasis> RGW now supports S3 Object Lock set of APIs allowing for a
+    WORM model for storing objects. 6 new APIs have been added put/get bucket
```

Comment:
What is the put/get option? Should it be <option>put/get</option> ?

Reply:
it's PUT or GET HTTP request
```suggestion
    WORM model for storing objects. 6 new APIs have been added PUT/GET bucket
```